packages/@aws-cdk/aws-rds/lib/serverless-cluster.ts

```diff
@@ -0,0 +1,544 @@
+   */
+  readonly securityGroups?: ec2.ISecurityGroup[];
+
+  /**
+   * The KMS key for storage encryption.
+   *
+   * @default - the default master key will be used for storage encryption
+   */
+  readonly storageEncryptionKey?: kms.IKey;
+
+  /**
+   * Additional parameters to pass to the database engine
+   *
+   * @default - no parameter group.
+   */
+  readonly parameterGroup?: IParameterGroup;
+}
+
+/**
+ * Create a clustered database with a given number of instances.
+ */
+export interface IServerlessCluster extends IResource, ec2.IConnectable, secretsmanager.ISecretAttachmentTarget {
+  /**
+   * Identifier of the cluster
+   */
+  readonly clusterIdentifier: string;
+
+  /**
+   * The endpoint to use for read/write operations
+   * @attribute EndpointAddress,EndpointPort
+   */
+  readonly clusterEndpoint: Endpoint;
+
+  /**
+   * Endpoint to use for load-balanced read-only operations.
+   * @attribute ReadEndpointAddress
+   */
+  readonly clusterReadEndpoint: Endpoint;
+}
+
+/**
+ * Properties that describe an existing cluster instance
+ */
+export interface ServerlessClusterAttributes {
+  /**
+   * Identifier for the cluster
+   */
+  readonly clusterIdentifier: string;
+
+  /**
+   * The database port
+   *
+   * @default - none
+   */
+  readonly port?: number;
+
+  /**
+   * The security groups of the database cluster
+   *
+   * @default - no security groups
+   */
+  readonly securityGroups?: ec2.ISecurityGroup[];
+
+  /**
+   * Cluster endpoint address
+   *
+   * @default - no endpoint address
+   */
+  readonly clusterEndpointAddress?: string;
+
+  /**
+   * Reader endpoint address
+   *
+   * @default - no reader address
+   */
+  readonly readerEndpointAddress?: string;
+}
+
+/**
+ * Aurora capacity units (ACUs).
+ * Each ACU is a combination of processing and memory capacity.
+ *
+ * @see https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/aurora-serverless.setting-capacity.html
+ * @see https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/aurora-serverless.how-it-works.html#aurora-serverless.architecture
+ */
+export enum AuroraCapacityUnit {
+  /**
+   * 1 Aurora Capacity Unit
+   */
+  ACU_1 = 1,
+
+  /**
+   * 2 Aurora Capacity Units
+   */
+  ACU_2 = 2,
+
+  /**
+   * 8 Aurora Capacity Units
+   */
+  ACU_8 = 8,
+
+  /**
+   * 16 Aurora Capacity Units
+   */
+  ACU_16 = 16,
+
+  /**
+   * 32 Aurora Capacity Units
+   */
+  ACU_32 = 32,
+
+  /**
+   * 64 Aurora Capacity Units
+   */
+  ACU_64 = 64,
+
+  /**
+   * 128 Aurora Capacity Units
+   */
+  ACU_128 = 128,
+
+  /**
+   * 192 Aurora Capacity Units
+   */
+  ACU_192 = 192,
+
+  /**
+   * 256 Aurora Capacity Units
+   */
+  ACU_256 = 256,
+
+  /**
+   * 384 Aurora Capacity Units
+   */
+  ACU_384 = 384
+}
+
+/**
+ * Options for configuring scaling on an Aurora Serverless cluster
+ */
+export interface ServerlessScalingOptions {
+  /**
+   * The minimum capacity for an Aurora serverless database cluster.
```

Comment:
`~` Nit - You've capitalized Serverless in most of the docstrings and README; might as well be consistent.

```suggestion
   * The minimum capacity for an Aurora Serverless database cluster.
```